webpack.config.js

```diff
@@ -22,8 +22,7 @@ if (DEV) {
   devtool = 'eval-source-map';
   loaders = ['react-hot'].concat(loaders);
   plugins = plugins.concat([
-    new webpack.HotModuleReplacementPlugin(),
-    new webpack.NoErrorsPlugin()
```

Comment:
Linter errors will not break the build. Actually having NoErrors for development does not make much sense anyway. Good for prod though.